### ISSUE TITLE
Installation error, loading script upgraded_depth_anything_v2.py

### ISSUE THREAD

**Issue by axymeus** [MackinationsAi/sd-webui-udav2#5]
*** Error loading script: upgraded_depth_anything_v2.py
    Traceback (most recent call last):
      File "D:\~diffuse\stable-diffusion-webui-1.7.0\modules\scripts.py", line 469, in load_scripts
        script_module = script_loading.load_module(scriptfile.path)
      File "D:\~diffuse\stable-diffusion-webui-1.7.0\modules\script_loading.py", line 10, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "D:\~diffuse\stable-diffusion-webui-1.7.0\extensions\sd-webui-udav2\scripts\upgraded_depth_anything_v2.py", line 21, in <module>
        'outdir_udav2': shared.OptionInfo(util.truncate_path(os.path.join(paths_internal.default_output_dir, 'depths', 'vis_image_depths')),
        'Output directory for UDAV2 images', component_args=shared.hide_dirs),
    AttributeError: module 'modules.util' has no attribute 'truncate_path'

When using Automatic1111 (1.7.0) internal extension installation.

**Comment by MackinationsAi** [comment]
Fixed w/ latest [update](https://github.com/MackinationsAi/sd-webui-udav2/commit/f948b6d6dbbb2a045f4e0e8a0ea421ae5ea2ee5d). Apologies for the delayed response, haven't had much free-time until now! I primarily use comfy, though occasionally when I do use a1111, I too use (v1.7.0), tried to install it internally from the extensions tab & got the same error as you. I've updated the script to prevent this error from occurring as well as several other errors I encountered too. Now everything is working smoothly @axymeus, thank you for bringing this to my attention! Simply update the udav2 extension inside a1111 & you'll be good to go.
